### DISCUSSION TITLE
v40.0

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#3733] [Release Discussions]
<h3>Important: Update manually after Feb 21!</h3>
  <p>Google Play / F-Droid will not update to the next major version v41 automatically because the app will ask for an additional permission. In v41, you'll be able to measure widths and heights with the camera! </p>
  <h3>New Quests</h3>
  <ul>
  <li><i>How does this road cross the barrier here?</i> (#3372, #3515), by @matkoniecz</li>
  <li><i>Can you pump gas yourself at this fuel station?</i> (#2822, #1827), by @naposm</li>
  <li><i>Does this place have air conditioning?</i> (#3641, #3127), by @coolultra1</li>
  <li><i>Are the opening hours signed for this place?</i>, asked at intervals for places that have previously been tagged as having no signed opening hours (#3682, #3130)</li>
  </ul>
  <h3>Quest enhancements</h3>
  <ul>
  <li>Allow answering whether there is a shoulder for each side individually (#3659)</li>
  <li>Allow answering whether the sidewalk is displayed separately on the map for each side individually</li>
  <li>When asking where a hydrant is located, show example pictures of pillar hydrants if it is a pillar hydrant (#3686, #3695, #3711), by @FloEdelmann</li>
  <li>Ask opening hours, place names and wheelchair accessibility for more amenities in the healthcare sector (#3710), by @FloEdelmann</li>
  <li>Other small tweaks in UI, quest filters and wording (#3593, #3726, #3657, #3727, #3719, #3693, #3712, #3722, #3716), thanks @matkoniecz, @peternewman, @smichel17, @arrival-spring</li>
  </ul>
  <h3>Fixes and improvements</h3>
  <ul>
  <li>More accurate conflict detection</li>
  <li>Fix a few rare crash issues</li>
  </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/streetcomplete/StreetComplete/releases/tag/v40.0'>v40.0</a>.</em>

**Comment by matkoniecz** [comment]
#3719 is not actually merged

**Reply by westnordost** [reply]
I know, but something was done as part/inspired by #3719
